plugins/modules/storage/netapp/sf_account_manager.py

```diff
@@ -21,7 +21,7 @@
   alternative: please use M(na_elementsw_account)
 short_description: Manage SolidFire accounts
 extends_documentation_fragment:
-- community.general.netapp.solidfire
```

Comment:
this module has been deprecated for 2 years.

I would rather not include it, as it is creating confusion.

Reply:
It has also been marked as deprecated with removal in Ansible 2.11. So it has to stay a bit longer, and will eventually be removed and replaced with a tombstone.